enterprise/templates/enterprise/emails/user_notification.html

```diff
@@ -1,15 +1,25 @@
 {% load i18n %}<html>
+
```

Comment:
Did you mean to change this file? These file changes appear to be why your tests are all failing.

Reply:
oh no, I had meant to restore this file but somehow have ended up changing it, this was just attempt to check if the test failures get fixed by restoring the file.. probably should try restoring it correctly first